src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, $width);
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
+    display: grid;
+    justify-items: center;
     align-items: center;
-    width: 200px;
+    width: $width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $gap;
   }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (max-width: 488px) {
```

Comment:
You should have 1 column under 488px, but you have 2 columns at 450px. Check your media queries for different width